### ISSUE TITLE
Kodi 16.1 generated-addons not rooted in WORKDIR

### ISSUE THREAD

**Issue by thespooler** [candrews/gentoo-kodi#1]
That causes the generated-addons to be unpacked as-is in $WORKDIR, not $S.
The tarball should include xbmc-16.1-Jarvis, like the 17.x does.

**Comment by candrews** [comment]
Sorry, what "causes the generated-addons to be unpacked as-is in $WORKDIR, not $S?"

I'm happy to fix this - just let me know what you're suggesting.

**Comment by thespooler** [comment]
Kodi 16.1 generated-addons is not rooted in WORKDIR.
Sorry about that, I now realise it was unclear because I skipped the subject.
The content of the tarball is incorrect for Kodi 16.1.
The 17.x ones contains xbmc-17.x-Krypton/xbmc/..., but not 16.1, wich is straight xbmc/...
The tarball just needs to have the folder wrapped, so xbmc should be under xbmc-16.1-Jarvis.

Hope this makes sense.

**Comment by candrews** [comment]
I've generated the 16.1 addons again. The contents now match the 16.0 generated addons.

I'm not sure what went wrong when I generated kodi-16.1-generated-addons.tar.xz... but kodi-16.1-generated-addons-r1.tar.xz should address all of the problems.

Note that this problem only occurred in kodi-16.1-generated-addons.tar.xz - the 17.0* archives appear to be fine.

The pull request for getting this fix into portage is https://github.com/gentoo/gentoo/pull/2875

See https://bugs.gentoo.org/show_bug.cgi?id=598058
